app/views/site/_expression_plots_plotly.html.erb

```diff
@@ -164,11 +165,11 @@ var expressionData = [];
     <% if @expression[:all][:x].length >= ClusterGroup::MAX_THRESHOLD %>
     title += ' (subsampled to <%= ClusterGroup::MAX_THRESHOLD %> cells)';
     <% end %>
-    var data = createTracesAndLayout(formatted_array, title, jitter_var);
+    var data = createTracesAndLayout(formatted_array, title, jitter_var, expressionLabel);
```

Comment:
Required parameters before optional, per convention:

```suggestion
    var data = createTracesAndLayout(formatted_array, title, expressionLabel, jitter_var);
```

This also reveals that we should use camel case in JavaScript wherever we can; the less snake case in JS the better.  I don't think that refactoring must happen now, though.